### PR TITLE
[libepoxy]Add support with unix.

### PR DESCRIPTION
**Pull request by JackBoosY**

### CODE REVIEW

File: ports/libepoxy/portfile.cmake [hunk 1]
```diff
@@ -9,12 +9,62 @@ vcpkg_from_github(
...
+
+if (VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_configure_meson(SOURCE_PATH ${SOURCE_PATH}
+        OPTIONS
+            -Denable-glx=no
+            -Denable-egl=no)
+    vcpkg_install_meson()
+    vcpkg_copy_pdbs()
+else()
+    message("libepoxy need to install autoreconf and libgles2-mesa-dev first.")
```
Comment (Rastaban) [inline ports/libepoxy/portfile.cmake]: You can use [find_program](https://cmake.org/cmake/help/v3.0/command/find_program.html) to check if these are installed and fail here..  You should also reword this to something like "autoreconf and libgles2-mesa-dev must be installed before libepoxy can build.  Install them with apt-get dh-autoreconf libgles2-mesa-dev" so it is clear that the user and not libepoxy needs to install them.

I will get them installed on the CI system.

**Comment by Rastaban** [comment]
I installed the recommended packages on the VMs and I am now seeing the following error.  Is there something else that needs to be installed?

```
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: ./configure --prefix=/ci/myagent/_work/1/s/buildtrees/libepoxy/make-build-x64-linux-release "--enable-x11=yes --enable-glx=yes --enable-egl=yes"
    Working Directory: /ci/myagent/_work/1/s/buildtrees/libepoxy/src/1.5.3-d7b9d32fcb
    Error code: 2
    See logs for more information:
      /ci/myagent/_work/1/s/buildtrees/libepoxy/config-x64-linux-out.log
      /ci/myagent/_work/1/s/buildtrees/libepoxy/config-x64-linux-err.log
```

[autoreconf-x64-linux-err.log](https://github.com/microsoft/vcpkg/files/3602918/autoreconf-x64-linux-err.log)
[autoreconf-x64-linux-out.log](https://github.com/microsoft/vcpkg/files/3602919/autoreconf-x64-linux-out.log)
[config-x64-linux-err.log](https://github.com/microsoft/vcpkg/files/3602920/config-x64-linux-err.log)
[config-x64-linux-out.log](https://github.com/microsoft/vcpkg/files/3602921/config-x64-linux-out.log)

**Comment by JackBoosY** [comment]
@Rastaban I will clone a new vcpkg and test build it on my machine.

**Comment by JackBoosY** [comment]
The file `configure` I generated does not contain `PKG_PROG_PKG_CONFIG`.
This is the content of lines 4702 to 4728:
```
# On IRIX 5.3, sys/types and inttypes.h are conflicting.
for ac_header in sys/types.h sys/stat.h stdlib.h string.h memory.h strings.h \
                  inttypes.h stdint.h unistd.h
do :
  as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
ac_fn_c_check_header_compile "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default
"
if eval test \"x\$"$as_ac_Header"\" = x"yes"; then :
  cat >>confdefs.h <<_ACEOF
#define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
_ACEOF

fi

done






ac_fn_c_check_decl "$LINENO" "__clang__" "ac_cv_have_decl___clang__" "$ac_includes_default"
if test "x$ac_cv_have_decl___clang__" = xyes; then :
  CLANGCC="yes"
else
  CLANGCC="no"
fi
```

Maybe the difference when using configure.ac to generate configure leads to the error.
And I cannot repro it on my VM.

**Comment by Rastaban** [comment]
It looks like I need to also install `pkg-config` on the CI machines.
